src/pages/Home/StudentHome.jsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react';
+import { useEffect, useState } from "react";
+import useAxiosPrivate from "../../hooks/useAxiosPrivate";
+import { Box, Typography } from "@mui/material";
+import { useNavigate } from "react-router-dom";
 
 const StudentHome = () => {
-    return (
-        <div>StudentHome</div>
-    );
+  const navigate = useNavigate();
+  const axiosPrivate = useAxiosPrivate();
+  const [cohorts, setCohorts] = useState([]);
+
+  useEffect(() => {
+    const fetchCohorts = async () => {
+      try {
+        const res = await axiosPrivate.get("/user/cohorts", {});
+        setCohorts(res.data.cohorts);
+      } catch (err) {
+        console.log(err);
+      }
+    };
+    fetchCohorts();
+  }, [axiosPrivate]);
+
+  if (cohorts.length === 1) {
+    navigate(`/student/cohort/${cohorts[0]._id}`, { state: cohorts[0] });
+  }
+
```

Comment:
If the student is not enrolled in a cohort, maybe redirecting them to a page that says something like "You are not enrolled in any class" would be helpful. Let me know your thoughts about this ☺